### PR TITLE
In the magazine

### PR DESCRIPTION
**Pull request by sumancrest0001**

### CODE REVIEW

**Comment by samgaco** [comment]
#### Hi @sumancrest0001 :vulcan_salute: 

#### Impressive work you have done :clap: some things to address before we can give you the approval:

* Correct the failing checks by the linters.
* **Improve the readme file,** you can add your contact details, a live demo of the site, inspire yourself by checking out some readme files templates. It is an important part of your project since everyone coming in will see it.
* **Correct the errors** given to you by the [validator](https://validator.w3.org) when uploading the index file.
* Some text is overflowing in this section for the small screen version:
morestories
* I suggest the use of the `<main> `tag for the dominant content of the `<body>` of a document
* Solve the conflicts on the pull request

### That's it. Your project looks amazing but please :pray: work on these changes and submit back for another code review once you are done :muscle: Regards!!

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Great job :+1: 
**Your page looks impressive, nicely done :tada:**
Great work implementing the suggested changes but there more things that we need to fix yet.

- [ ]  There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

- [ ] We need to fix the errors found by the linter. Linting is important to improve the overall quality of your code, it also gives a professional look in your repository.

Please kindly make these changes and submit for code review again :smile: 
Happy coding :computer:

File: css/main.css [hunk 1]
```diff
@@ -1,98 +1,826 @@
...
+  }
+
+  #current-date {
+    display: none;
+  }
+
+  .page-header {
+    width: 100%;
+    padding-top: 0.5rem;
+	  padding-bottom: 0.5rem;
```
Comment (bolah2009) [inline css/main.css:697]: #### `REQUIRED`

Please, fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many Code Editor can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

Screenshot of how to format code using VS Code

**Review by bolah2009 — APPROVED** [review]
# It's a :+1: from me.

### I would strongly recommend fixing the linter error before merging to master. Use code editor for formatting your code (see comment).

Also, format readme file properly.

Good luck with your next project.

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**